app/Http/Controllers/BeatmapsController.php

```diff
@@ -76,7 +76,7 @@ public function scores($id)
                 }
 
                 $results = [
-                    'scores' => json_collection($query->visibleUsers()->forListing(), 'Score', ['beatmap', 'user', 'user.country']),
+                    'scores' => json_collection($query->visibleUsers()->forListing(), 'Score', ['beatmap', 'user', 'user.country', 'user.cover']),
```

Comment:
userProfileCustomization needs to be preloaded otherwise this will cause 49 additional queries.